resources/less/hakija.less

```diff
@@ -47,6 +47,50 @@
 .application__banner-wrapper-section-valid-img {
   margin-right: 5px;
 }
+.application__invalid-field-status {
+  font-size: 12px;
+  margin-top: 2px;
+  text-align: center;
+  width: 190px;
+  position: relative;
+  margin-right: 20px;
+}
+.application__invalid-field-status-title {
+  cursor: pointer;
+}
+.application__invalid-fields {
+  z-index: 1; /* On mobile (narrow screen) section line is above this container without this */
+  position: absolute;
+  top: 26px;
+  left: 17px;
+  text-align: start;
+  background-color: white;
+  color: @pale-blue;
+  min-width: 170px;
+  padding: 11px;
+  box-shadow: 1px 2px 5px #b3b3b3;
```

Comment:
Generally, I think it'd be nice if we could extract _all_ the color definitions to variables... (maybe not a thing for the PR though)